includes/abstracts/abstract-wc-order.php

```diff
@@ -272,7 +272,7 @@ public function update() {
 	 * Delete data from the database.
 	 * @since 2.7.0
 	 */
-	public function delete() {
+	public function delete( $force_delete = false ) {
 		wp_delete_post( $this->get_id() );
```

Comment:
I think you mentioned these would be adjusted later but just leaving a note that this is not using the force_delete prop or new methods.